dev-ruby/ipaddr/ipaddr-1.2.5.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2022 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby30 ruby31"
+RUBY_FAKEGEM_EXTRADOC="README.md"
+
+inherit ruby-fakegem
+
+DESCRIPTION="An IPv4/IPv6 address manipulation library for Ruby"
+HOMEPAGE="https://github.com/ruby/ipaddr"
+SRC_URI="https://github.com/ruby/ipaddr/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Ruby"
+SLOT="0"
```

Comment:
For new packages I've started to use $(ver_cut 1) as the default value which is less confusing when we need to slot a new major version. Not a deal-breaker for adding it.